README.md

```diff
@@ -21,7 +21,7 @@ Indeed, the filter grammar and parser and [`pydantic`](https://pydantic-docs.hel
 
 ## Prerequisites
 
-Environment where AiiDA is installed.  
+Environment where AiiDA is installed.
```

Comment:
This trailing white-space should _not_ be removed (see the argument I have added to the respective hook - why did you remove the original exception there??)

Reply:
Ah sorry, I simply didn't realize it is used here.

Reply:
It's not just here - please add the extra argument I have put in my other comment above

Reply:
That's true and using `--markdown-linebreak-ext=md` instead of excluding the `README.md` require double space to represent a newline. Changed.

Reply:
There were double spaces there originally as well.